pkg/scheduler/scheduler.go

```diff
@@ -242,31 +559,40 @@ func initPolicyFromConfigMap(client clientset.Interface, policyRef *kubeschedule
 func NewFromConfig(config *factory.Config) *Scheduler {
```

Comment:
Are there any remaining callers to this function? Can we delete it yet?

Reply:
There are still some callers in the tests. I will delete it after replacing all of them.